Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -90,6 +91,16 @@ async function run() {
         kubectlCli = new kubernetescli(kubeconfigfilePath);
         kubectlCli.login();
     }
+  
+    try {
+        const kubelogin = new Kubelogin(helmutil.getTaskTempDir());
+        if (kubelogin.isAvailable()) {
+          tl.debug('Kubelogin is installed. Converting kubeconfig.');
+          await kubelogin.login(tl.getInput('azureSubscriptionEndpoint', false));
+        }
+    } catch (err) {
+        tl.warning(tl.loc('KubeloginFailed', err));
```

Comment:
>warning [](http://example.com/codeflow?start=11&length=7)

same feedback on error vs warning as Tasks/KubernetesV1/src/clusterconnection.ts

Reply:
change to debug message here too